openshift/04-deploymentconfig-template.yml

```diff
@@ -98,6 +100,80 @@ objects:
               cpu: 250m
               memory: 250M
           terminationMessagePath: /dev/termination-log
+        - env:
+          - name: RAILS_LOG_TO_STDOUT
+            value: "true"
+          - name: RAILS_ENV
+            value: production
+          - name: DATABASE_URL
+            valueFrom:
+              secretKeyRef:
+                key: DATABASE_URL
+                name: zync
+          - name: SECRET_KEY_BASE
+            valueFrom:
+              secretKeyRef:
+                key: SECRET_KEY_BASE
+                name: zync
+          - name: BUGSNAG_API_KEY
+            valueFrom:
+              secretKeyRef:
+                key: BUGSNAG_API_KEY
+                name: zync
+          - name: SKYLIGHT_AUTHENTICATION
+            valueFrom:
+              secretKeyRef:
+                key: SKYLIGHT_AUTHENTICATION
+                name: zync
+          - name: ZYNC_AUTHENTICATION_TOKEN
+            valueFrom:
+              secretKeyRef:
+                key: ZYNC_AUTHENTICATION_TOKEN
+                name: zync
+          - name: RAILS_LOG_LEVEL
+            value: info
+          args:
+          - que
+          - --worker-count
+          - 10
+          image: zync:latest
+          imagePullPolicy: Always
+          livenessProbe:
+            failureThreshold: 3
+            httpGet:
+              path: /status/live
+              port: 8080
+              scheme: HTTP
+            initialDelaySeconds: 10
+            periodSeconds: 10
+            successThreshold: 1
+            timeoutSeconds: 60
+          name: metrics
```

Comment:
que is processing jobs in background, not exposing metrics

Reply:
done on 031e3fe1281af2f2942a74e57e6aa37fb372f302